alerts.tf

```diff
@@ -146,3 +146,26 @@ module "wa-task-deletion-failure-alert" {
   enabled                    = var.enable-wa-task-management-api-task-deletion-failure-alert
   common_tags                = var.common_tags
 }
+
+module "wa-message-readiness-check-failure-alert" {
+  source                     = "git@github.com:hmcts/cnp-module-metric-alert"
+  location                   = var.location
+
+  app_insights_name          = "wa-${var.env}"
+
+  alert_name                 = "wa-case-event-handler-message-readiness-check-failure-alert"
+  alert_desc                 = "Alert when case event handler message readiness check fails and auto restart of pod happens"
+  app_insights_query         = "traces | where message contains 'Liveness check failed' or message contains 'Readiness check failed'"
```

Comment:
This is probably ok to start with as we want to monitor it more closely.  However, once we are confident with the checks the idea would be to spot instances where we have a lot of failures in a relatively short space of time.  So maybe we can create a Jira to make an update to this along those lines?

Reply:
Can approve once Jira is created.

Reply:
Thanks Ravi, Created Jira : [RWA-3138](https://tools.hmcts.net/jira/browse/RWA-3138)